chapters/simple_exec.md

```diff
@@ -63,15 +59,19 @@ To help users quickly use these patterns, we reserved the following keywords for
 | Reserved Name | Description |
 | --- | --- |
 | `_clear` | Clear request body from a message |
+| `concat` | Concat all embeddings into one, grouped by ``doc.id`` |
+| `eval_pr` | Evaluate ``precision`` and ``recall`` |
+| `_index` | A simple compound indexer of ``NumpyIndexer`` and ``BinaryPbIndexer`` |
 | `_logforward` | Like `_pass`, but print the message |
 | `_pass` | Pass the message to the downstream |
-| `_merge_topk` | Merge the top-k search results (both doc and chunk level) of all collected messages, often used in the tail of a Pod |
-| `_merge_topk_chunks` | Merge the top-k search results (chunk level only) of all collected messages, often used in the tail of a Pod |
-| `_merge_topk_docs` | Merge the top-k search results (doc level only) of all collected messages, often used in the tail of a Pod |
-
+| `_merge_chunks` | Merges chunks from requests |
+| `_merge_matches` | Merges matches from requests |
+| `_merge_matches_topk` | Merge the top-k search results at match level |
+| `_merge_eval` | Merge all evaluations into one, grouped by ``doc.id`` |
+| `_merge_root` | Merge results at root level |
 
 ## How to use built-in Simple Executors?
 
 You can directly use this executor by specifying `--uses=_clear`, or use it in `--uses-after` after collecting results from replicas.
```

Comment:
or use it in `--uses-after` after collecting results from replicas. 
That sentence is not grammatically correct?